lib/senders/email_service.js

```diff
@@ -34,13 +42,36 @@ module.exports = (config) => {
     }
 
     request(options, function(err, res, body) {
+      if (! err && res.statusCode >= 400) {
+        err = marshallError(res.statusCode, body)
+      }
       cb(err, {
         messageId: body && body.messageId,
-        message: err ? err.message : body.message
+        message: body && body.message
```

Comment:
Since `err` is already being sent to the callback, there's no need to include `err.message` again here. And stopping it from over-riding `body.message` means we can log both error fields, as per https://github.com/mozilla/fxa-auth-server/pull/2566/files#r208200050.